pkg/sensors/tracing/generictracepoint.go

```diff
@@ -351,6 +354,7 @@ func createGenericTracepoint(
 		customHandler: customHandler,
 		message:       msgField,
 		tags:          tagsField,
+		hasFDInstall:  selectorsHaveFDInstall(conf.Selectors),
```

Comment:
Do we really need the extra field? Can we just run the check when we check the field?

Reply:
ah yes we can do that, I mimicked the kprobe stuff but this is not needed here indeed.

Reply:
because we no longer have access to the spec in the kprobe side, so we need that, but here it's simpler.

Reply:
I see, thanks!

Reply:
https://github.com/cilium/tetragon/compare/a44a1fa6995cdad90fd43ece109646df66c56cbd..2631f9b91d87247770531dfc1582665ebf19935c